docs/zh/engines/table-engines/mergetree-family/replacingmergetree.md

```diff
@@ -1,4 +1,4 @@
-# 更换麦树 {#replacingmergetree}
+# 替换合并树 {#replacingmergetree}
```

Comment:
Maybe nouns should not be translated? ReplacingMergeTree is better?

Reply:
It sounds like better. but I can't tolerate the wrong translated version.